src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/LoginPageBase.java

```diff
@@ -24,4 +28,24 @@ public LoginPageBase(WebDriver driver) {
 
 	public abstract CarinaDescriptionPageBase login();
 
+	public abstract boolean isNamePresented() throws Exception;
+
+	public abstract boolean isPassPresented() throws Exception;
+
+	public abstract boolean isNameEntered(String name) throws Exception;
+
+	public abstract boolean isPassEntered(String pass) throws Exception;
+
+	public abstract boolean isGenderPresented() throws Exception;
+
+	public abstract boolean isPageOpened();
+
+	public abstract boolean isMaleSelected() throws Exception;
+
+	public abstract boolean isPrivacyChecked() throws Exception;
```

Comment:
will be better if you create your exception with some message about not implemented method
and remove throws